vision_msgs/Detection3DArray detections
---
robot_manipulation/PackingPlan plan
